python/tk_multi_launchapp/base_launcher.py

```diff
@@ -13,6 +13,7 @@
 
 import sgtk
 from sgtk import TankError
+from sgtk.deploy import util
```

Comment:
this is now `sgtk.util` - but it's also an undocumented part of the core API - ideally we shouldn't call it.

Instead of calling these methods, i think it might be easier to refactor `_sort_versions` to call LooseVersion directly (http://epydoc.sourceforge.net/stdlib/distutils.version.LooseVersion-class.html) 

There is some good stuff here - http://stackoverflow.com/questions/12255554/sort-versions-in-python - the LooseVersion object has a `__cmp__` so you can sort the objects which might lead to quite an elegant solution!